src/main/kotlin/org/jetbrains/bio/big/BigWig.kt

```diff
@@ -80,24 +96,36 @@ class BigWigFile private constructor(input: RomBuffer,
             when (types[type - 1]) {
```

Comment:
Do we have explicit mention that all the sections are monotonous? Otherwise previous code worked, and this one will fail.